syntax = "proto3";

package roboapp;

import "buf/validate/validate.proto";

message CameraSwitchMessage {
  int32 camera_id = 1 [(buf.validate.field).int32 = {
    gte: 0
    lte: 2
  }];
}
